lib/dom.js

```diff
@@ -204,7 +204,7 @@ define([], function () {
             if (!(cell && cell.header)) { return; }
             //HACK for IE10, does not like literal enum
             enu = cell.header['enum'];
-            if (self.dispatchEvent('beforebeginedit', {cell: cell})) { return false; }
+            if (self.dispatchEvent('beforebeginedit', {cell: cell, NativeEvent})) { return false; }
```

Comment:
Can you change this to es5 syntax?  There's no transpiler hooked up to build and IE11 is a target.  Otherwise, looks great.

Reply:
Sure thing.